docs/15-deploy-to-localstack.md

```diff
@@ -0,0 +1,72 @@
+Building and deploying Sleeper locally
+======================================
+
+You can also run a localstack container locally and deploy an instance of Sleeper to it. This deployment method has
```

Comment:
Docke, LocalStack and Sleeper should be capitalised, as they're proper nouns. I'd probably also match LocalStack's own capitalisation. There are a few places in this file they're in lower case or capitalised differently.